packages/amplify-java-template-provider/src/index.ts

```diff
@@ -0,0 +1,18 @@
+import { FunctionTemplateContributorFactory } from 'amplify-function-plugin-interface';
+
+import { provideHelloWorld } from './providers/helloWorldProvider';
+
+export const functionTemplateContributorFactory: FunctionTemplateContributorFactory = context => {
+  return {
+    contribute: selection => {
```

Comment:
rebase with latest change to master (the selection is now a request object with the selection string as a property)